tests/sweep_framework/sweeps/matmul/short/matmul_default_sharded.py

```diff
@@ -17,71 +17,71 @@
     "default": {
         "matmul_specs": [
             # mcast 2d
-            """(
-            (2, 3),
-            (1600, 224, 896),
-            False,
-            dict(core_grid=ttnn.CoreGrid(y=5, x=7), strategy=ttnn.ShardStrategy.BLOCK),
-            None,
-        )""",
+            (
+                (2, 3),
+                (1600, 224, 896),
+                False,
+                dict(core_grid=ttnn.CoreGrid(y=5, x=7), strategy=ttnn.ShardStrategy.BLOCK),
```

Comment:
This is not going to work when you try to run these tests. This should be in the same format as matmul/short/matmul_user_program_config.py without the ttnn types in tuples. TTNN types nested within tuples won't be serialized properly.

Reply:
Thanks. Updating.